packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -126,13 +134,13 @@ describe('NetworkController', () => {
                 infuraProjectId: 'infura-project-id',
               },
               ({ controller }) => {
-                const fakeInfuraProvider = buildFakeInfuraProvider();
-                createInfuraProviderMock.mockReturnValue(fakeInfuraProvider);
-                const fakeInfuraSubprovider = buildFakeInfuraSubprovider();
-                SubproviderMock.mockReturnValue(fakeInfuraSubprovider);
-                const fakeMetamaskProvider = buildFakeMetamaskProvider();
-                createMetamaskProviderMock.mockReturnValue(
-                  fakeMetamaskProvider,
+                expect(controller.state.isCustomNetwork).toBe(true);
+                const fakeJsonRpcClient = buildFakeInfuraClient();
```

Comment:
something I noticed was that the networkController tests as they exist on main - only infura providers are created, even when the test looks like its testing standardProvider. 

Could rename buildFakeInfuraClient to buildFakeClient, since its the only one ever used in these tests. Based on the title of the tests, I interpreted this as a mistake in the underlying tests, and left the name as fakeJsonRpcClient (since thats what it *should* be creating).